ibm/service/vpc/resource_ibm_is_backup_policy.go

```diff
@@ -0,0 +1,407 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func ResourceIBMIsBackupPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMIsBackupPolicyCreate,
+		ReadContext:   resourceIBMIsBackupPolicyRead,
+		UpdateContext: resourceIBMIsBackupPolicyUpdate,
+		DeleteContext: resourceIBMIsBackupPolicyDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"match_resource_types": &schema.Schema{
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Computed:    true,
+				Set:         schema.HashString,
+				Description: "A resource type this backup policy applies to. Resources that have both a matching type and a matching user tag will be subject to the backup policy.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"match_user_tags": &schema.Schema{
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "The user tags this backup policy applies to. Resources that have both a matching user tag and a matching type will be subject to the backup policy.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Set:         schema.HashString,
+			},
+			"name": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validate.InvokeValidator("ibm_is_backup_policy", "name"),
+				Description:  "The user-defined name for this backup policy. Names must be unique within the region this backup policy resides in. If unspecified, the name will be a hyphenated list of randomly-selected words.",
+			},
+			"resource_group": &schema.Schema{
```

Comment:
Follow the same standard of setting resource_group similar to other VPC resources

Reply:
done.